editor/place/views.py

```diff
@@ -375,6 +375,8 @@ def edit_place():
             apply_change(wof_doc, request.form, "edtf:deprecated")
             apply_change(wof_doc, request.form, "edtf:inception")
             apply_change(wof_doc, request.form, "edtf:superseded")
+            apply_change(wof_doc, request.form, "wof:superseded_by")
```

Comment:
These will also need a `list_of_int` validator created and used as a 4th argument to this function. See `list_of_str` examples below. The function will need to be created up above, too.

```suggestion
            apply_change(wof_doc, request.form, "wof:superseded_by", list_of_int)
```